src/tools/illink/src/ILLink.RoslynAnalyzer/ILLink.RoslynAnalyzer.csproj

```diff
@@ -28,6 +31,7 @@
       <PrivateAssets>all</PrivateAssets>
       <ExcludeAssets>contentfiles</ExcludeAssets> <!-- We include our own copy of the ClosedAttribute to work in source build -->
     </PackageReference>
+    <PackageReference Include="System.Reflection.Metadata" Version="$(SystemReflectionMetadataVersion)" />
```

Comment:
This change seems to break the repo build. See https://github.com/dotnet/runtime/issues/106321

Reply:
Yeah, an analyzer should only be using the version of SRM that roslyn provides for it.

It could be possible for an analyzer to carry a newer version of SRM, but it would need to be copied/deployed with the analyzer - and we'd need to make sure to never try to exchange SRM types with the compiler (I don't *think* any SRM types are part of roslyn's public API but I can't be certain).